README.md

```diff
@@ -1616,6 +1616,10 @@ Here is the simplest example of a chat room (i.e. mediator) with users (i.e. col
 First of all, we have the mediator i.e. the chat room
 
 ```php
+interface ChatRoomMediator {
```

Comment:
Could you please [move this curly bracket to new line](http://www.php-fig.org/psr/psr-2/)?